components/list/list.js

```diff
@@ -55,6 +56,9 @@ class List extends PageableMixin(SelectionMixin(LitElement)) {
 			slot[name="pager"]::slotted(*) {
 				margin-top: 12px;
 			}
+			section {
+				display: unset;
```

Comment:
Curious to know more about this - why the default `display` value isn't good...?

Reply:
Seems to be interacting with `sticky` in a way I wasn't expecting, and causes this visual-diff failure with the default `block` display: https://visual-diff.d2l.dev/screenshots/BrightspaceUI/core/list/2022-11-30-20.38.52.695-a648e674d047ed0eae6d71f1aaa92c5ccd113b79-report.html
Setting this to `unset` (or `contents` or `inline`) seems to fix it locally, will see if it does in CI as well

Reply:
Yeah, that seemed to fix it.  I'm guessing it was somehow messing with the intersection observer, I can play around a bit

Reply:
I'm going to have a quick look into why section would be messing up sticky...

Reply:
> I'm going to have a quick look into why section would be messing up sticky...

Awesome, that would be great!  I'm not finding anything obvious, there's some posts about `overflow` being a problem or some saying "Change `display` to `initial` or `inline`" but they don't explain _why_

Reply:
😂 This is my life with sticky. Yes, overflow's the huge main problem that always rears its head.

That's not what's going on here though - it's about having *any* parent element (even a div). But there are other cases where that's not a problem - still trying to get to the bottom of it...

Reply:
Okay so yes, it 100% makes sense that putting a wrapping div around a sticky element would break it. Here's the Two Basic Rules of Sticky (™️ Erik Koopmans):

1. A sticky element is only free to move within the confines of its nearest block-level ancestor
2. A sticky element sticks relative to its nearest "scrolling container" (e.g. element with any overflow defined)

That's it. There's no breaking of those rules allowed, period (when using CSS sticky). So a wrapping `section` (with block display) applies to item 1 - and since the wrapper is the same size as the sticky element, it has nowhere to move.

I was getting confused because I've been dealing so much with Rule 2, I forgot about Rule 1 😛.